include/deployer.bash

```diff
@@ -201,6 +202,20 @@ doctor() {
     generate_uaa_check_diff verbose
 }
 
+network-doctor() {
+    ping 8.8.8.8 -c 1 &> /dev/null
+    if [[ "$?" -ne "0" ]]; then
+        error "Could not resolve 8.8.8.8."
```

Comment:
probably "Couldnt reach" or "Couldnt ping" but "resolve is missleading" as there is no dns involved yet